server/plugin/command_test.go

```diff
@@ -239,3 +271,65 @@ func TestCheckConflictingFeatures(t *testing.T) {
 		})
 	}
 }
+
+func TestExecuteCommand(t *testing.T) {
+	tests := map[string]struct {
+		commandArgs    *model.CommandArgs
+		expectedMsg    string
+		SetupMockStore func(*mocks.MockKvStore)
+	}{
+		"about command": {
+			commandArgs:    &model.CommandArgs{Command: "/github about"},
+			expectedMsg:    "GitHub version",
+			SetupMockStore: func(mks *mocks.MockKvStore) {},
+		},
+
+		"help command": {
+			commandArgs: &model.CommandArgs{Command: "/github help", ChannelId: "test-channelId", RootId: "test-rootId", UserId: "test-userId"},
+			expectedMsg: "###### Mattermost GitHub Plugin - Slash Command Help\n",
+			SetupMockStore: func(mks *mocks.MockKvStore) {
+				mks.EXPECT().Get(gomock.Any(), gomock.Any()).DoAndReturn(func(key string, value interface{}) error {
+					// Cast the value to the appropriate type and updated it
+					if userInfoPtr, ok := value.(**GitHubUserInfo); ok {
+						*userInfoPtr = &GitHubUserInfo{
+							// Mock user info data
+							Token: &oauth2.Token{
+								AccessToken: "ycbODW-BWbNBGfF7ac4T5RL5ruNm5BChCXgbkY1bWHqMt80JTkLsicQwo8de3tqfqlfMaglpgjqGOmSHeGp0dA==",
```

Comment:
Are we using an actual token here if not we should replace it with a mock value

Reply:
It is a mock value only, but of a specific size i.e. 88 bytes, so using a random string here

Reply:
We should replace it with a value like "mockToken-mockToken-MockToken........". This will clear the confusion for the reviewer